src/main/java/com/assure/movie/controller/MemberController.java

```diff
@@ -1,20 +1,26 @@
 package com.assure.movie.controller;
 
 import com.assure.movie.common.converter.MemberConverter;
+import com.assure.movie.common.response.CreatedResponse;
 import com.assure.movie.dto.MemberDTO;
 import com.assure.movie.model.domain.Member;
 import com.assure.movie.service.MemberService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.util.List;
 
 /**
  * @author Israel Yasis
  */
 @RestController
+@Api(tags = "Members")
 public class MemberController extends ApiController {
```

Comment:
It shouldn't be good to use the _@RequestMapping("members")_ here?

Reply:
If add a request mapping in the class,  every request mapping in my controller will have the "member" in the path. it can be a better option for my controller.

Reply:
That is right, since all endpoints in this Controller should be related to a Member operation, having that definition as a class definition is a good approach.